CDCgov/prime-reportstream
# Add Telecom to FHIR path

This PR adds the patient's phone number to the fhir path.

Test Steps:
1. Send fhir file with telecom field to the api.
2. Make sure the phone number is displayed

## Changes
- Edits to patient-contact.yml to make the filter work

## Checklist

### Testing

## Linked Issues
- ETOR - add telecom
